cli/script/command-executor.ts

```diff
@@ -224,6 +238,75 @@ function appRename(command: cli.IAppRenameCommand): Promise<void> {
         });
 }
 
+function appTransfer(command: cli.IAppTransferCommand): Promise<void> {
+    throwForInvalidEmail(command.email);
+
+    return getAppId(command.appName)
+        .then((appId: string): Promise<void> => {
+            throwForInvalidAppId(appId, command.appName);
+
+            return confirm()
+                .then((wasConfirmed: boolean): Promise<void> => {
+                    if (wasConfirmed) {
+                        return sdk.transferApp(appId, command.email)
+                            .then((): void => {
+                                log("Successfully transferred the ownership of \"" + command.appName + "\" to the account with email \"" + command.email + "\".");
```

Comment:
Look at other methods like appRename(...) and deploymentAdd(...); when displaying a name in a message, add its type.

"Successfully transferred the ownership of app ..." (I added "app" there.)

This feedback also applies to the messages in addCollaborator(...) and removeCollaborator.